api/examples/reconfigure_vm.md

```diff
@@ -0,0 +1,55 @@
+---
+---
+
+### Reconfigure VM
+
+#### Request:
+
+    POST /api/requests
+
+``` json
+{
+  "action" : "create",
+  "options": {
+    "request_type": "vm_reconfigure",
+    "src_ids":[ "2" ],
+    "vm_memory": "8192",
+    "number_of_cpus": 2,
+    "cores_per_socket": 2
```

Comment:
NOTE there are other options such as adding/removing disks, nics, cdroms I'll follow-up with those once I validate the format

Reply:
This has always driven me nuts, but the menu for the api actually lives in the manageiq.org repo here: https://github.com/ManageIQ/manageiq.org/blob/master/site/_data/menus/api_menu.yml

See also https://github.com/ManageIQ/manageiq.org/issues/814

Reply:
Was this a reply to https://github.com/ManageIQ/manageiq-documentation/pull/1624#issuecomment-995176330 ?

Yeah okay just confirming, I'll put a PR in to manageiq.org as well

Reply:
Gah yeah...sorry

Reply:
https://github.com/ManageIQ/manageiq.org/pull/1042